docs/operating/binary-index-header.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Binary index-header
+type: docs
+menu: operating
+slug: /binary-index-header.md
+---
+
+# Binary index-header
+
+In order to query series inside blocks from object storage, Store Gateway has to know certain initial info about each block such as:
+
+- symbols table to unintern string values
+- postings offset for posting lookup
+
+In order to achieve so, on startup for each block `index-header` is built from pieces of original block's index and stored on disk.
```

Comment:
Let's clearly mention that store does not UPLOAD any of this. It's a common misconception.

Reply:
Done. What do you think?